src/current/cockroachcloud/production-checklist.md

```diff
@@ -6,35 +6,78 @@ cloud: true
 docs_area: deploy
 ---
 
-This page provides important recommendations for CockroachDB {{ site.data.products.cloud }} production deployments.
+Before deploying CockroachDB {{ site.data.products.cloud }} in production, it is important to understand the Shared Responsibility Model that delineates the responsibilities of Cockroach Labs and the customer in managing CockroachDB {{ site.data.products.cloud }} clusters. 
 
-## Follow the SQL Best Practices
+Under the Shared Responsibility Model, Cockroach Labs is responsible for the following tasks: 
 
-To ensure optimal SQL performance for your CockroachDB {{ site.data.products.cloud }} cluster, follow the best practices described in the [SQL Performance Best Practices](https://www.cockroachlabs.com/docs/{{site.current_cloud_version}}/performance-best-practices-overview) guide.
+- Cluster and cloud service availability and reliability
+- Maintenance and security of hardware and operating systems
+- Database upgrades and security patches 
+- Daily and hourly cluster backups 
 
-## Use a pool of persistent connections
+The customer is responsible for the following tasks:
 
-Creating the appropriate size pool of connections is critical to gaining maximum performance in an application. Too few connections in the pool will result in high latency as each operation waits for a connection to open up. But adding too many connections to the pool can also result in high latency as each connection thread is being run in parallel by the system. The time it takes for many threads to complete in parallel is typically higher than the time it takes a smaller number of threads to run sequentially.
+- Sizing the workload correctly
+- Scaling clusters based on workload
+- Ensuring sufficient disk, compute, and memory capacity for each cluster
+- Monitoring cluster health and application performance
```

Comment:
I don't feel strongly here, but are people able to delineate between "cluster health" and "hardware/os help" listed above? I feel like honestly this is fine as worded to start but more curious what you think here. App perf for sure is their responsibility. I am SO HAPPY we're explicitly stating this. SRE == us, DBA + App Dev == you

Reply:
I believe there is room for misinterpretation here. Some customers will ask "don't you manage the cluster as it's a managed service?"

Reply:
Maybe add a point below - something like:
Ensuring that the workload is distributed appropriately across the nodes of the cluster

Reply:
@alistair-roacher IIRC, we want the users to know that even if we manage the cluster, they are still responsible for monitoring the cluster health and app performance, right?

Reply:
@Amruta-Ranade - Yes absolutely. We will monitor the cluster from an availability perspective but the customer should monitor all other aspects of cluster health.